settings/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "main": "../dist/settings.js",
+  "types": "../dist/settings.d.ts"
+}
```

Comment:
Interesting, do we need this?

Can't we just use [the new exports spec](https://nodejs.org/api/packages.html#packages_package_entry_points)?

Reply:
It's making settings available as an import `nexus-prisma/settings`. I'm not familiar with the exports spec limitations. Let me know which solution you prefer and I will implement it.

Reply:
Please give the `exports` spec a shot. It is not very complex for our use-case here. Also doing nothing should still technically work, no? Is this just for some kind of discoverability? I am curious about this pattern for other reasons (a documentation tool I am working on, that needs to support whatever patterns are used in the wild) so if there's a blog post or something that goes into more detail happy to read it.

Reply:
20 days ago the TypeScript team acknowledged they are not expecting to ship it soon https://github.com/microsoft/TypeScript/issues/33079#issuecomment-825392940

Reply:
It's not needed for this PR. Just remove it.